app/javascript/packs/violet_analytics.js

```diff
@@ -0,0 +1,61 @@
+$(document).on("turbo:load", () => {
```

Comment:
https://nikean.org story platform uses non-turbo navigation but the site uses turbo. We should accommodate both cases